examples/iOS/Backend/Sources/Planbok/Dependencies/Keychain.swift

```diff
@@ -1,56 +1,20 @@
 //
 //  File.swift
-//  
+//
 //
 //  Created by Alexander Cyon on 2024-02-15.
 //
 
 @_exported import KeychainAccess
 import Sargon
-import SargonUniFFI
+import ComposableArchitecture
 
-extension DependencyValues {
-  /// A dependency that exposes an ``Keychain.Dependency`` value that you can use to read and
-  /// write to `Keychain`.
-  public var keychain: Keychain.Dependency {
-	get { self[Keychain.Dependency.self] }
-	set { self[Keychain.Dependency.self] = newValue }
-  }
-}
 
 extension Keychain: @unchecked Sendable {}
-extension Keychain: SecureStorage {
-	@DependencyClient
-	public struct Dependency: DependencyKey {
-		
-		public let loadData: @Sendable (SecureStorageKey) throws -> Data?
-		public let saveData: @Sendable (SecureStorageKey, Data) throws -> Void
-		public let deleteDataForKey: @Sendable (SecureStorageKey) throws -> Void
-		
-		public static func with(keychain: Keychain) -> Self {
-			Self.init(
-				loadData: keychain.loadData(key:),
-				saveData: keychain.saveData(key:data:),
-				deleteDataForKey: keychain.deleteDataForKey(key:)
-			)
-		}
-		public static let liveValue = Self.with(keychain: .shared)
-		public static var testValue: Self {
-			final class Ephemeral {
-				var dict = Dictionary<SecureStorageKey, Data>()
-				init() {}
-			}
-			let ephemeral = Ephemeral()
-			return Self(
-				loadData: { ephemeral.dict[$0] },
-				saveData: { ephemeral.dict[$0] = $1 },
-				deleteDataForKey: { ephemeral.dict[$0] = nil }
-			)
-		}
-	}
-	
+extension Keychain: SecureStorageDriver {
+
 	public static let shared = Keychain(service: "works.rdx.planbok")
-	
+
```

Comment:
Mmm we should probably go via `SargonOS.shared.secureStorage().load` then, probably in a TCA "client", since loading sensitive information such as mnemonics require auth, we cannot and should not do it with TCA SharedReader, so a "client" for `load` makes sense.

Reply:
Yes, and that's exactly what we are doing actually 🙂 We don't need the `shared` Keychain instance on iOS side, if you delete it it would still compile as it isn't used.